alu_u32/src/mul/stark.rs

```diff
@@ -57,13 +57,15 @@ fn pi_m<const N: usize, AB: AirBuilder>(
 ) -> AB::Expr {
     iproduct!(0..N, 0..N)
         .filter(|(i, j)| i + j < N)
-        .map(|(i, j)| base[i + j].clone() * input_1[i] * input_2[j])
+        .map(|(i, j)| base[i + j].clone() * input_1[3 - i] * input_2[3 - j])
```

Comment:
Just to check if I understand, seems currently `base` is LE while `input`s are BE, but the latter we'll switch later?

Reply:
Correct, I switched the endianness of `base` here to fix some bugs. Still unsure if we should change words to BE.